### ISSUE TITLE
The temperature input for the slider does not have it's role explicitly defined programatically (i.e. role="slider")  

### ISSUE THREAD

**Issue by Tanvez** [Harvard-University-iCommons/chainlit#25]
| IMPACT      | URL          | ISSUE DESCRIPTION   | WCAG CRITERIA | TESTING METHOD | SCREENSHOT or SNIPPET | ACTION |
| :----:          |    :----:      |      :----:                        |  :----:                    | :----:                        | :----:                                     |  :----:    | 
| low    | n/a         | The temperature input for the slider does not have it's role explicitly defined programatically (i.e. role="slider")  | 1.3.1 Info and Relationships - A | Manual - Chrome | n/a | From the context most Browsers will understand this is a slider but it would be better to be explicit and add the role="slider" attribute.
